pkg/rpcmiddleware/discover/discover.go

```diff
@@ -21,9 +21,11 @@ import (
 	"net"
 	"strings"
 
```

Comment:
Nit: We don't usually put empty lines between stdlib imports.

Reply:
`gofumports` does that